src/Actions/MediaAction.php

```diff
@@ -84,7 +84,7 @@ protected function setUp(): void
                                 $set('height', $dimensions[1]);
                             }
                         })
-                        ->saveUploadedFileUsing(function (BaseFileUpload $component, TemporaryUploadedFile $file, callable $set) {
+                        ->saveUploadedFileUsing($component->getSaveUploadedFileUsing() ?: function (BaseFileUpload $component, TemporaryUploadedFile $file, callable $set) {
```

Comment:
Hi @awcodes,
It works because at line 57, `->form(function (TiptapEditor $component) {`, allows access to the $component variable. The getSaveUploadedFileUsing method returns a callback, and during execution, Filament automatically injects variables into this callback.